include/llbuild/Basic/Utility.h

```diff
@@ -0,0 +1,35 @@
+//===- Utility.h ------------------------------------------------*- C++ -*-===//
```

Comment:
Rather than making this `Utility`, which is a little too generic for my tastes, I suggest using `ShellUtility`.